tests/integration_test.rs

```diff
@@ -26,24 +26,45 @@ use up_rust::{
 use uprotocol_zenoh::UPClientZenoh;
 use zenoh::config::Config;
 
-fn create_utransport_uuri() -> UUri {
-    UUri {
-        entity: Some(UEntity {
-            name: "body.access".to_string(),
-            version_major: Some(1),
-            id: Some(1234),
+fn create_utransport_uuri(index: u8) -> UUri {
```

Comment:
Is the idea that you'd in the future keep adding more `UUri`s in here for testing purposes and you want to have a single factory function for creating them?

Reply:
You're right. That's my intention.
Feel free to give me your opinions about this. I'm still wondering whether there is a more elegant way to do so.